### PR TITLE
API docs fixed, dependencies updated

### PR DESCRIPTION
**Pull request by kailash**
- API docs fixed
- pom dependencies are updated. version is maintained as used in all other servers

### CODE REVIEW

File: docs/openapi.yaml [hunk 1]
```diff
@@ -1,15 +1,25 @@
...
+
 consumes:
-  - application/json
+  - "application/json"
+  - "text/plain"
+
+
 info:
   title: IUDX Data Ingestion Server APIs
   version: 3.5.0
```
Comment (abhi4578) [inline docs/openapi.yaml:20]: can version be changed to 4.0.0 here and all api servers? @kailash

Comment (kailash) [reply docs/openapi.yaml:20]: yes, will change this in all API docs

**Comment by karun-singh** [comment]
retest this please

**Comment by kailash** [comment]
closes #18 also

**Pull request merged**